systemrequirements.adoc

```diff
@@ -115,20 +115,19 @@ a|Administrator Computers +
  Other configurations may work.
 |The following operating system/browser combinations are supported:
 
-// TODO this looks well out of date.  How about Windows 10?  Edge browser?
-Windows XP with required updates, Vista, Windows 7, or Windows 8 with
+Windows Vista SP 2, Windows 7, Windows 8 or Windows 10 with
 one of the following:
 
-* Internet Explorer 8.0 and higher
-** IE8.x for XP
-** IE9.x and higher for Vista/Windows 7
-** IE10 and higher for Windows 8
+* Microsoft support is only available for Internet Explorer 11 or Microsoft Edge
+** IE11 and higher for Vista SP 2/Windows 7
+** IE11 and higher for Windows 8
+** IE11 or Microsoft Edge (Supported since ZCS 8.6 P4 and above) for Windows 10 
 * The latest stable release of:
 ** Firefox
 ** Safari
 ** Google Chrome
 
-Mac OS X 10.5, 10.6, 10.7, or 10.8 with one of the following:
+Mac OS X 10.5, 10.6, 10.7, 10.8, 10.9, 10.10, or 10.11 with one of the following:
```

Comment:
My current Mac OS Sierra is 10.12.6 - it would be strange not to support this (saying perhaps 10.12).  Same comment applies below.

Reply:
You are right Gren, let me add it